packages/frontend/app/components/ui/LiquidityDialog.tsx

```diff
@@ -0,0 +1,62 @@
+import { Dialog } from '@headlessui/react'
```

Comment:
What should go into `components/ui` vs just `components/`? I'm just thinking whether there is a better spot for this component specifically

Reply:
Oh, I misplaced this. Basically, `ui` is similar to `shared` - most reusable code blocks and the components that comprise the UI of the applications (buttons, inputs, etc...).